pkg/webservice/handlers_test.go

```diff
@@ -1358,6 +1376,34 @@ func TestUsersAndGroupsResourceUsage(t *testing.T) {
 	assert.Equal(t, usersResourceUsageDao[0].Queues.ResourceUsage.String(),
 		resources.NewResourceFromMap(map[string]resources.Quantity{siCommon.CPU: 1}).String())
 
+	// Test existed user query
+	req, err = http.NewRequest("GET", "/ws/v1/partition/default/usage/user/testuser", strings.NewReader(""))
```

Comment:
Is there a way to assert " /ws/v1/partition/default/usage/user/ " or without specific user or group name being passed? In this case, there is no need to call the corresponding method in ugm module and "user or group name is missing or mandatory param is missing" error message.

Reply:
If you can check and make changes for the above, we can start merging after code changes. Code coverage too complains this specific path.